pydoctor/templatewriter/summary.py

```diff
@@ -26,13 +26,26 @@ def moduleSummary(module: model.Module, page_url: str) -> Tag:
         r(class_='private')
     if not isinstance(module, model.Package):
         return r
-    contents = [m for m in module.contents.values()
-                if isinstance(m, model.Module) and m.isVisible]
+    contents = list(module.submodules())
     if not contents:
         return r
-    ul = tags.ul()
+
     def fullName(obj: model.Documentable) -> str:
         return obj.fullName()
+
+    if len(contents) > 50 and not any(any(s.submodules()) for s in contents):
+        # If there are more than 50 modules and no submodule has
+        # further submodules we use a more compact presentation.
+        pass
+        div = tags.div(class_='submodules')
+        for m in sorted(contents, key=fullName):
+            div(tags.code(tags.a(m.name, href=m.url)))
```

Comment:
Please use epydoc2stan.taglink to create links to objects in the system.

Reply:
Why? Of the three `if`s in epydoc2stan.taglink none will ever apply:

* `if not o.isVisible`: not the case because we filter modules for visibility before
* `if label is None:` we would need to specify the label since we don't want the full name
* `if url.startswith(page_url + '#'):` also will never apply

So using it seems quite pointless to me.

Reply:
Because we want to centralize the link generation at only one place such that we can add custom css for internal vs external links without too much hazard. See #468